### ISSUE TITLE
HLS Audio support on Android

### ISSUE THREAD

**Issue by SatoshiInoue** [expo/expo#1187]
Hi, 
I'm looking for a solution to play HLS audio and tested Expo.Audio.Sound out of this project as suggested in the official docs: https://github.com/expo/playlist-example

However, I'm getting an error if I try to play an HLS audio stream URL on Android with the following error message:

Possible Unhanded Promise Rejection
...Error: java.lang.illegalStateException: mSimpleExoPlayer is null!

In short, is HLS supported by the Expo AV APIs as of today?
(I was able to play mp3 & IceCast 2.4 on both iOS & Android, but I could play HLS only on iOS)

**Comment by sjchmiela** [comment]
Sorry, at the moment HLS is not supported by the Expo AV player. ☹️ 

However, we'll try to ship it in the next release. 🙂

**Comment by SatoshiInoue** [comment]
No problem. Looks like it's happening soon:)

**Comment by SatoshiInoue** [comment]
@sjchmiela I'd assume this has been already released in SDK 25?

**Comment by terribleben** [comment]
Yep! See [the release notes](https://blog.expo.io/expo-sdk-v25-0-0-is-now-available-714d10a8c3f7).

**Comment by SatoshiInoue** [comment]
Terrific. Thank you, guys. I've tested playlist-example app for myself and I was able to stream an HLS audio URL!
